js/app.js

```diff
@@ -597,32 +600,43 @@
 				}
 			} else {
 				OCA.SpreedMe.app.videoWasEnabledAtLeastOnce = false;
-				OCA.SpreedMe.app.disableVideo();
+				OCA.SpreedMe.app.availableVideo();
 			}
 		},
 		enableAudio: function() {
 			OCA.SpreedMe.webrtc.unmute();
-			$('#mute').attr('data-original-title', 'Mute audio')
+			$('#mute').attr('data-original-title', t('spreed', 'Mute audio'))
 				.removeClass('audio-disabled icon-audio-off-white')
 				.addClass('icon-audio-white');
 
 			OCA.SpreedMe.app.audioDisabled = false;
 		},
 		disableAudio: function() {
 			OCA.SpreedMe.webrtc.mute();
-			$('#mute').attr('data-original-title', 'Enable audio')
+			$('#mute').attr('data-original-title', t('spreed', 'Enable audio'))
 				.addClass('audio-disabled icon-audio-off-white')
 				.removeClass('icon-audio-white');
 
 			OCA.SpreedMe.app.audioDisabled = true;
 		},
+		availableVideo: function() {
+			var $hideVideoButton = $('#hideVideo');
+
+			if (OCA.SpreedMe.webrtc.localStreams !== null) {
```

Comment:
Did you actually test this? `OCA.SpreedMe.webrtc.localStreams` should be a list in which the last entry is the new local stream. You then need to check if this stream contains an audio and/or video track.